content/asp.net/getting-started/mvc.md

```diff
@@ -1,20 +1,122 @@
+
+## Project Structure and Conventions
+In our solution created from the default project template there are a few points to note related to the folder layout and the conventions of the framework. In the root of the project we find some key classes, like the ``program.cs`` file with our application entry point, and the ``startup.cs`` class we were discussing above. There is a ``project_readme.html`` with information about the libraries used in the template and some helpful links related to MVC development.  
+
+You will find the familiar ``project.json`` file, some default application settings in the ``appsettings.json`` file, as well configuration hints for bundling and minification.  To enable our application to run on [Kestrel behind IIS Express](https://docs.asp.net/en/latest/fundamentals/servers.html#servers-and-commands), there is a ``web.config`` file with settings for our [host](hosts.md).
+
+Each project will have a folder called `wwwroot`, configured by [default](https://docs.asp.net/en/latest/intro.html#web-root) to be the base URL for the root of our website. The server will attempt to serve up requests from this folder and its subfolders when the static file middleware is enabled, such as for image and JavaScript requests.
+
+There are two other meaningful folders in the project called "Controllers" and "Views", which contain those respective concepts as files you include in your project. For each controller that returns views from one or more actions, you'll also typically find a folder under "Views" named the same as the controller containing all of its associated views.
+
+When we talk about "MVC" we're talking about Models, Views and Controllers.  Let's talk about those compoents in greater detail next.
+
+## Models
+Models are the properties associated with some kind of business concept. A contact, invoice or movie pass all have some type of representation assosciated with them. In MVC Core they are simply represented as classes that you create in your project or import via a library dependency.
+
+We use models to carry the data we need to our users. Models are typically created or fetched from a service in our Controllers and are passed to our Views for rendering.  
+
+You will learn more about models when we look at [Returning Data or Views from Controllers](results.md) later in this tutorial.
+
+## Views
+Views are the mechanism by which we generate the HTML needed to render our web pages for the user's browser. While we don't exclusively write HTML in our Views it is a large part of it. Instead, we use the Razor view engine and (Razor syntax)(https://docs.asp.net/en/latest/mvc/views/razor.html) to _output_ HTML.  
```

Comment:
Wrong kind of braces, should be: `[Razor syntax](https://docs.asp.net/en/latest/mvc/views/razor.html)`.